scripts/Shape.js

```diff
@@ -0,0 +1,14 @@
+function Shape(fret1,fret2,fret3,fret4,fret5,fret6,fret7,fret8,fret9,fret10,fret11,fret12) {
```

Comment:
Instead of passing in all of these frets as separate arguments could we simply pass in an array of frets, and then have one property that is that array? Example:

```javascript
function Shape(frets) {
  this.frets = frets;
}

var myShape = new Shape([125,100,75,50,25,0, 1,26,52,77,102,126]);
```

That way when `drawLines` is called, we don't even need to loop through them, we can just pass the array directly. This would also make sure it's flexible enough if there does end up being a shape that isn't 12 frets.